tgui/.gitignore

```diff
@@ -9,3 +9,4 @@ package-lock.json
 !/.yarn/plugins
 !/.yarn/releases
 !/.yarn/sdks
+!/.yarn/lock.yml
```

Comment:
This doesn't seem to get created, but excluding from being gitgnored in case it appears.